modules/ROOT/pages/salesforce-to-twilio.adoc

```diff
@@ -0,0 +1,454 @@
+
+Visually, the process should look like this:
+
+image::salesforce-to-twilio-flowchart.png[]
+
+Fortunately, you know just the tool to help you carry out this plan: Flow Designer!
+
+== Before you begin
+
+* Sign up for a 30-day trial of xref::https://anypoint.mulesoft.com/login/signup[MuleSoft Anypoint Platform], if you do not already have access to
+* Sign up for xref::https://developer.salesforce.com/signup[a free developer account from Salesforce]. You'll need your credentials for this account, as well as your security token.
+* Sign up for xref::https://www.twilio.com/try-twilio[a free Twilio developer account]. You'll need a trial phone number that Twilio provides for your account. You'll also need your account SID and authentication token.
+
+== Capture opportunities that are won
+You need to ensure that your application is triggered as soon as the status of an Opportunity in Salesforce is changed to ‘Closed Won’. To do this, you specify in Salesforce which fields to send and our criteria for triggering the application. You can specify all of this with an SOQL query. After you build the query, you can set up a Push Topic that can serve as the trigger for your MuleSoft application.
+
+. Navigate to the xref::https://workbench.developerforce.com/query.php[SOQL query builder]. (You have to log in with your Salesforce credentials.)
+. Select *Opportunity* as the Object.
+. Select *Amount*, *CloseDate*, *Id*, and *Name* from the *Fields* menu.
+. In the *Filter results by* fields, add `StageName = ‘Closed Won’`.
+. Test the query by using the *Query* button. You should get a list of all closed Opportunities. Your query should look like this : `SELECT Amount,CloseDate,Id,Name FROM Opportunity WHERE StageName = 'Closed Won'`
++
+image::soql-query-screen-1.png[]
+
+. Copy the query to your clipboard.
+. Navigate to the Streaming Push topics page from the *query* menu at the top of the workbench.
+. Create a new Push Topic by selecting the option *Create New*.
+. Name the Push Topic `ClosedOpps` and paste in the query that you built.
+. Save the Push Topic.
++
+image::streaming-push-topics.png[]
+
+Now, every time the query criterion is met, Salesforce will push an event into the `ClosedOpps` Push Topic.
+
+== Create a project in Flow Designer
+
+When you are working in Flow Designer, you create your Mule applications in projects. A project can contain only one Mule application.
+
+A Mule application consists of one or more _flows_. According to the MuleSoft glossary, a flow is "a simple, flexible mechanism that enables orchestration of services using the sophisticated message flow capabilities of Mule runtime engine. Also referred to as service orchestration." In Flow Designer, what this means is a sequence of _cards_ that represent core components in Mule Runtime or the operations of connectors. (Cards can represent a couple of other things, too; however, for this tutorial, you need know only about those two.)
```

Comment:
@rddheath i'm struggling with the line that says "In Flow Designer, what this means is a sequence of _cards_ that represent core components in Mule Runtime or the operations of connectors." 
i think it could be confusing for new readers. What do you think about saying 

"In Flow Designer, what this means is a sequence of _cards_ that represent core components (i.e. loops, conditionals, transformations etc) or connectors to Systems and APIs (i.e. Salesforce, SAP, Google Docs)"